src/plugins/intel_gpu/src/runtime/ocl/ocl_ext.hpp

```diff
@@ -925,21 +925,21 @@ class UsmMemory {
 
     void allocateHost(size_t size) {
         cl_int error = CL_SUCCESS;
-        _allocate(_usmHelper.allocate_host(nullptr, size, 0, &error));
+        _allocate(_usmHelper.allocate_host(nullptr, ((size-1)/65536 + 1)*65536, 65536, &error));
```

Comment:
1. please add named const for alignment
2. you can probably use some simple helper here which does alignment
so ideally it should look like `_allocate(_usmHelper.allocate_host(nullptr, align_to(size, alignment), alignment, &error));`

Reply:
sorry for the confusion. this is not for merge. But for experiment.